### PR TITLE
fix(numbers_tag): use <dl>

### PR DESCRIPTION
**Pull request by verarojman**
- Use items instead of object for numbers tag
- Bump version
- Update readme
- Use dl tag for numbers

### CODE REVIEW

File: lib/jekyll-plugin-platoniq-journal/numbers_tag.rb [hunk 1]
```diff
@@ -47,21 +47,21 @@ def include_file_path
     def render_all
       output = []
 
-      output << %(<ul class="pj-numbers">)
+      output << %(<dl class="pj-numbers">)
```
Comment (agustibr) [inline lib/jekyll-plugin-platoniq-journal/numbers_tag.rb]: should this be a `<div>`?
```suggestion
      output << %(<div class="pj-numbers">)
```

File: lib/jekyll-plugin-platoniq-journal/numbers_tag.rb [hunk 2]
```diff
@@ -47,21 +47,21 @@ def include_file_path
...
       jdata["items"].each do |item|
         output << <<~NUM
-          <li>
-            <strong>
-              #{item["number"]}
-            </strong>
-            <span class="pj-number--description">
-              <span class="pj-number--icon">#{@icon}</span>
+          <div class="pj-numbers--item">
```
Comment (agustibr) [inline lib/jekyll-plugin-platoniq-journal/numbers_tag.rb]: .. and this a `<dl>`?
```suggestion
          <dl class="pj-numbers--item">
```

**Review by agustibr — COMMENTED** [review]
great! just a doubt about having `<dl><div>` so the div inside the description-list 😁

File: lib/jekyll-plugin-platoniq-journal/numbers_tag.rb (hunk 2 above)
Comment (verarojman) [reply lib/jekyll-plugin-platoniq-journal/numbers_tag.rb]: I thought about that, but I'm not sure what's best... So if you think it's better to have many `dl` elements nested inside a `div`, I can change it.

Comment (agustibr) [reply lib/jekyll-plugin-platoniq-journal/numbers_tag.rb]: Yes, could you change it please, in my opinion it makes more sense 😄

**Pull request merged**